File: backend/backend/Models/Address.cs
namespace backend.Models;

public class Address : BaseEntity
{
    public string Country { get; set; }
    public string Place { get; set; }
    public string Street { get; set; }
    public string Number { get; set; }
    public string PostalCode { get; set; }
    public virtual User User { get; set; }
}

File: frontend/src/model/entities/Address.ts
export class Address {
    constructor(
        public country: string,
        public place: string,
        public postalCode: string,
        public street: string,
        public number: string,
    ) {
    }

    public toString() {
        return this.street + " " + this.number + ", " + this.place + ", " + this.postalCode + ", " + this.country;
    }
}
